### PR TITLE
meta-mender-intel: initial template files for Minnowboard

### PR DESCRIPTION
**Pull request by mirzak**
Signed-off-by: Mirza Krak <mirza.krak@northern.tech>

### CODE REVIEW

**Comment by mirzak** [comment]
@estenberg, this is what the Minnowboard would look like here. Based on your instructions in Mender Hub.

I have build tested this.

File: meta-mender-intel/scripts/manifest-intel.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+  <manifest>
+  <default sync-j="4" revision="sumo"/>
+
+  <remote fetch="git://git.yoctoproject.org"        name="yocto"/>
+  <remote fetch="git://github.com/linux-sunxi"      name="sunxi"/>
```
Comment (estenberg) [inline meta-mender-intel/scripts/manifest-intel.xml]: Is this needed?

File: meta-mender-intel/scripts/manifest-intel.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <remote fetch="git://git.openembedded.org"        name="oe"/>
```
Comment (estenberg) [inline meta-mender-intel/scripts/manifest-intel.xml]: Why do we need openembedded? Isn't yoctoproject enough?

File: meta-mender-intel/scripts/manifest-intel.xml (hunk 1 above)
Comment (mirzak) [reply meta-mender-intel/scripts/manifest-intel.xml]: No, leftover from copy/paste

File: meta-mender-intel/scripts/manifest-intel.xml (hunk 2 above)
Comment (mirzak) [reply meta-mender-intel/scripts/manifest-intel.xml]: Also a left over from copy/paste. Poky is enough and meta-intel does not have any other dependencies.

**Comment by mirzak** [comment]
> I suppose the Mender Hub Minnowboard page should be updated?

Yes once this is merged and validated we should update  Minnowboard page.

> Moreover, should we update the template as well?

Yes, I think it is a good idea.

**Comment by mirzak** [comment]
Updated, also note one more addition to the `local.conf.append`. I had some issues building `core-image-base` and this tries to address that.

**Comment by estenberg** [comment]
Looks good, thanks for fixing @mirzak !
Would you mind updating the minnowboard topic in hub as well?

**Comment by mirzak** [comment]
I would not mind :)

**Comment by mirzak** [comment]
Updated the minnowboard topic

**Pull request merged**